dotnet/aspnetcore31/functionApp/Application/SampleFunctionApp/SampleFunctionApp.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
```

Comment:
is this forced by .net core?

Reply:
No, .NET Core 3.1 wasn't supported in Azure functions V2, thus upgrading to V3. https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions#languages